tests/observability/metrics/utils.py

```diff
@@ -1283,6 +1285,103 @@ def get_metric_labels_non_empty_value(prometheus: Prometheus, metric_name: str)
     return {}
 
 
+def get_pod_memory_stats(admin_client: DynamicClient, hco_namespace: str, pod_prefix: str) -> int:
+    return int(
+        bitmath.Byte(
+            int(
+                get_pod_by_name_prefix(
+                    dyn_client=admin_client,
+                    pod_prefix=pod_prefix,
+                    namespace=hco_namespace,
+                ).execute(command=RSS_MEMORY_COMMAND)
+            )
+        )
+    )
+
+
+def get_highest_memory_usage_virt_api_pod_tuple(hco_namespace: str) -> tuple[str, int]:
+    virt_api_with_highest_memory_usage = (
+        run_command(
+            command=shlex.split(
+                f"oc adm top pod -n {hco_namespace} --sort-by memory --no-headers -l kubevirt.io=virt-api"
+            )
+        )[1]
+        .strip()
+        .split("\n")[1:]
+    )[0].split()
+    return (
+        virt_api_with_highest_memory_usage[0],
+        int(bitmath.parse_string_unsafe(virt_api_with_highest_memory_usage[2]).Byte),
+    )
+
+
+def get_pod_requested_memory(hco_namespace: str, admin_client: DynamicClient, pod_prefix: str) -> float:
+    return float(
+        bitmath.parse_string_unsafe(
+            get_pod_by_name_prefix(
+                dyn_client=admin_client,
+                pod_prefix=pod_prefix,
+                namespace=hco_namespace,
+            )
+            .instance.spec.containers[0]
+            .resources.requests.memory
+        ).Byte
+    )
+
+
+def expected_kubevirt_memory_delta_from_requested_bytes(
+    hco_namespace: str, admin_client: DynamicClient, rss: bool
+) -> float:
+    highest_memory_usage_virt_api_pod = get_highest_memory_usage_virt_api_pod_tuple(hco_namespace=hco_namespace)
+    virt_api_pod_name = highest_memory_usage_virt_api_pod[0]
```

Comment:
will be clearer to unpack to  variable names:
```suggestion
    pod_name, pod_memory = get_highest_memory_usage_virt_api_pod_tuple(hco_namespace=hco_namespace)

```